tests/providers/azure/lib/mutelist/azure_mutelist_test.py

```diff
@@ -66,3 +66,37 @@ def test_is_finding_muted(self):
         finding.subscription = "subscription_1"
 
         assert mutelist.is_finding_muted(finding)
+
+    def test_mute_finding(self):
+        # Mutelist
+        mutelist_content = {
+            "Accounts": {
+                "subscription_1": {
+                    "Checks": {
+                        "check_test": {
+                            "Regions": ["*"],
+                            "Resources": ["test_resource"],
+                        }
+                    }
+                }
+            }
+        }
+
+        mutelist = AzureMutelist(mutelist_content=mutelist_content)
+
+        finding = MagicMock
```

Comment:
Why are you not using a `Finding`? We have a cool generator for this.

Reply:
The same for the rest of the tests.

Reply:
Done